src/gui/gui.h

```diff
@@ -69,10 +70,14 @@ class DissolveWindow : public QMainWindow
     QLabel *restartFileIndicator_;
     // Label for iteration number
     QLabel *iterationLabel_;
-    // Label for simulation ETA (when using RunFor)
-    QLabel *etaLabel_;
+    // Label for simulation timer (ETA if RunFor, time elapsed if Run)
+    QLabel *timerLabel_;
+    // Timer label default text
+    static constexpr const char *timerText_ = "Idle";
     // General status indicator and label
     QLabel *statusIndicator_, *statusLabel_;
+    // Main loop iteration timer
+    Timer elapsedTimer_ = Timer(false);
```

Comment:
Prefer brace initialisation:
```suggestion
    // Main loop iteration timer
    Timer elapsedTimer_{Timer(false)};
```